index.html

```diff
@@ -853,105 +753,80 @@ <h4>The <dfn>ThingActionInit</dfn> dictionary</h4>
       <section data-dfn-for="ThingEventInit">
         <h4>The <dfn>ThingEventInit</dfn> dictionary</h4>
         <pre class="idl">
-          dictionary ThingEventInit {
-            DOMString name;
-            sequence&lt;SemanticType&gt; semanticTypes;
-            ThingDescription dataDescription;
+          dictionary ThingEventInit: SemanticAnnotations {
+            required DOMString name;
+            ValueType dataDescription;
```

Comment:
Align with TD draft, name it `outputData`?

Reply:
`outputData` is a bad name, usually events have "data" in JS. This API is for developers.